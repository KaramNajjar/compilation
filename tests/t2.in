123 123
obj		
<<

/kiwi
-22.

/asisi (I know I'm not right,\
 but it doesn't make me /wrong) /vitamin 00001110010101.7 /C false
			
				>>
endobj