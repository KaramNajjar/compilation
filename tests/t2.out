Producing Obj
Producing Body
Producing Dict
Producing KVList
Producing KV
Producing Exp
Finished producing Exp
Finished producing KV
Producing KVList
Producing KV
Producing Exp
Finished producing Exp
Finished producing KV
Producing KVList
Producing KV
Producing Exp
Finished producing Exp
Finished producing KV
Producing KVList
Producing KV
Producing Exp
Finished producing Exp
Finished producing KV
Producing KVList
Finished producing KVList
Finished producing KVList
Finished producing KVList
Finished producing KVList
Finished producing KVList
Finished producing Dict
Finished producing Body
Finished producing Obj
ObjNode {BodyNode [Dictionary Node <<KVListNode <Key : Value(Expr), KVListNode <Key : Value(Expr), KVListNode <Key : Value(Expr), KVListNode <Key : Value(Expr), KVListNode <> End KVListNode> End KVListNode> End KVListNode> End KVListNode> End KVListNode>> End Dictionary] End BodyNode} End ObjNode
